spec/models/post_spec.rb

```diff
@@ -1,5 +1,66 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  text = 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.
+  Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+  Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim.
+  Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a,
+  venenatis vitae, justo. Nullam dictum felis eu pede mollis pretium. Integer tincidunt. Cras dapibus.
+  Vivamus elementum semper nisi. Aenean vulputate eleifend tellus. Aenean leo ligula, porttitor eu,
+  consequat vitae, eleifend ac, enim. Aliquam lorem ante, dapibus in, viverra quis, feugiat a, tellus.
+  Phasellus viverra nulla ut metus varius laoreet. Quisque rutrum. Aenean imperdiet. Etiam ultricies
+   nisi vel augue. Curabitur ullamcorper ultricies nisi. Nam eget dui. Etiam rhoncus. Maecenas tempus,
+   tellus eget condimentum rhoncus, sem quam semper libero, sit amet adipiscing sem neque sed ipsum.
+   Nam quam nunc, blandit vel, luctus pulvinar, hendrerit id, lorem. Maecenas nec odio et ante tincidunt tempus.
+   Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt.
+  Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna. Sed consequat, leo eget bibendum sodales,
+  augue velit cursus nunc, quis gravida magna mi a libero. Fusce vulputate eleifend sapien. Vestibulum purus quam,
+  scelerisque ut, mollis sed, nonummy id, metus. Nullam accumsan lorem in dui. Cras ultricies mi eu turpis hendrerit
+  fringilla. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; In ac dui quis mi
+  consectetuer'
+
+  before :each do
+    User.create(id: 34, name: 'Leo', PostsCounter: 0)
+  end
+
+  subject do
+    Post.new(id: nil, user_id: 1, title: 'Anything', text: 'hhh crazy', CommentsCounter: 2, LikesCounter: 1)
+  end
+
+  before { subject.save }
+
+  it 'Post was successfully created.' do
+    subject.text = text
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a title' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a text' do
+    subject.text = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid with LikesCounter negative' do
+    subject.LikesCounter = 5
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid with LikesCounter not integer' do
+    subject.LikesCounter = 7
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid with CommentsCounter negative' do
+    subject.CommentsCounter = 5
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid with CommentsCounter not integer' do
+    subject.CommentsCounter = 7
+    expect(subject).to_not be_valid
+  end
 end
```

Comment:
- [ ] Kindly make sure that your Post model contains validation logic then write tests for your validation logic. Check the example bellow 

```rb
class Post < ApplicationRecord


  validates :title, :text, presence: true
  validates :title, presence: { message: 'Title must not be blank' },
                    length: { maximum: 250, too_long: 'Title must not exceed 250 characters' }
  validates_numericality_of :comments_counter, only_integer: true, greater_than_or_equal: 0
  validates_numericality_of :like_counter, only_integer: true, greater_than_or_equal: 0


end
```
- [ ] It would be better also to render validation errors in the correspondent view.